test/unit/api/test_dist_analytics.py

```diff
@@ -0,0 +1,127 @@
+from api.app.analysis import get_geojson_from_data_api, get_geojson_request_for_data_api
+from api.app.query import create_gadm_dist_query
+import re
+import pytest
+
+def test_create_gadm_adm2_dist_query_no_intersection():
+    query = create_gadm_dist_query(["IDN", "24", "9"], [])
+    
+    assert query == strip_extra_whitespace("""
+        SELECT country, region, subregion, alert_date, alert_confidence AS confidence, SUM(count) AS value
+        FROM 's3://gfw-data-lake/umd_glad_dist_alerts/parquet/gadm_dist_alerts.parquet'
+        WHERE country = 'IDN' AND region = 24 AND subregion = 9
+        GROUP BY country, region, subregion, alert_date, alert_confidence
+        ORDER BY country, region, subregion, alert_date, alert_confidence
+    """)
```

Comment:
I think you could break these up into tests that focus on `SELECT`, `FROM`, `WHERE`, etc.
It would be easier to see what part was broken or needed to change in the future.

Reply:
So just do the same operation 4 times, but assert on a different part of the statement?